### DISCUSSION TITLE
Deploy-button not available except on main branch?

### DISCUSSION THREAD

**Discussion by terminalnode** [woodpecker-ci/woodpecker#3625] [Ask a Question] [answered]
Once a PR has been merged to the main branch I have the deploy-button available from the workflows, but for workflows not running on the main branch I don't. Is there a setting to enable this?

My use-case is that sometimes I want to do a test-deploy to our develop environment, which includes building an image of my current branch. If not, is there some other way I can manually trigger a workflow from a feature branch?

**Comment by qwerty287** [comment, marked as answer]
This has been temporarily disabled in 2.4.1 for security reasons, but will be available again in the next version (however, there it must be enabled in the settings).
